superset/charts/commands/exceptions.py

```diff
@@ -78,13 +78,21 @@ class ChartDeleteFailedError(DeleteFailedError):
     message = _("Chart could not be deleted.")
 
 
+class ChartDeleteFailedReportsExistError(ChartDeleteFailedError):
+    message = _("There are associated alerts or reports associated")
```

Comment:
The word "associated" is used twice here. Would "The chart has associated alerts or reports" make sense?

Reply:
fixed